packages/containers/src/ConfirmDialog/showHideConfirmDialog.js

```diff
@@ -0,0 +1,17 @@
+export function showConfirmDialog(state, action) {
+	const path = ['ConfirmDialog', 'ConfirmDialog'];
+	const newState = Object.assign({}, state);
+	newState.cmf.components = state.cmf.components.setIn(path, action.confirmDialogConf);
+	return newState;
+}
+
+export function hideConfirmDialog(state) {
+	//hiding the modal
+	const path = ['ConfirmDialog', 'ConfirmDialog', 'show'];
+	const newState = { ...state };
+	newState.cmf.components = state.cmf.components.setIn(path, false);
+	return newState;
+}
+
```

Comment:
Your 2 apis are not consistent.
`showConfirmDialog` should show the dialog and `hideConfirmDialog` should hide it.

They should use a common code (from `src/state.js` ? (see my global review comment)) and only customise the value (true/false).